pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -206,3 +206,17 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	}
 	return utilerrors.NewAggregate(allErrors)
 }
+
+// ValidateKubeletLoggingConfiguration validate kubelet logging part config
+func ValidateKubeletLoggingConfiguration(kc *kubeletconfig.KubeletConfiguration) error {
```

Comment:
Please check code to validate config couple of lines above. it uses logs.ValidateLoggingConfiguration function instead.

Reply:
I cann't get the point.
I see logOption.Validate() call ValidateLoggingConfiguration too, but ValidateKubeletLoggingConfiguration is called ealier than ValidateKubeletConfiguration.

Reply:
We should replace logOptions.Validate() with `ValidateLoggingConfiguration`